content/blog/2021-02-18-introducing-the-@-modifier.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Introducing the '@' Modifier
+created_at: 2021-02-18
+kind: article
+author_name: Ganesh Vernekar
+---
+
+Currently, the `topk()` query only makes sense as an instant query where you get exactly `k` results, but when you run it as a range query, you can get much more than `k` results since every step is evaluated independently. This `@` modifier lets you fix the ranking for all the steps in a range query.
```

Comment:
```suggestion
Have you ever selected the top 10 time series for something, but instead of 10 you got 100? If yes, this one is for you. Let me walk you through what the underlying problem is and how I fixed it.

Currently, the `topk()` query only makes sense as an instant query where you get exactly `k` results, but when you run it as a range query, you can get much more than `k` results since every step is evaluated independently. This `@` modifier lets you fix the ranking for all the steps in a range query.
```